Makefile.am

```diff
@@ -157,6 +157,7 @@ WEBPACK_PACKAGES = \
 	selinux \
 	shell \
 	sosreport \
+	ssh \
```

Comment:
Sometimes we want this functionality without the dashboard. EI the kubernetes container. That's why I pulled out it. But we can always just mess with it in the container setup scripts if you think that's better.